### PR TITLE
feat: :sparkles: add question detail drawer

### PR DESCRIPTION
**Pull request by syoung125**
adjust table width

### CODE REVIEW

File: src/components/question /detail/drawer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+
+import { ApplicantSection, ItemSection, QuestionSection } from '../sections';
+
+import { IQuestion } from '@src/interfaces/Question/IQuestion';
+
+export type QuestionDetailDrawerContentProps = {
+  question: IQuestion;
+};
```
Comment (greatSumini) [inline src/components/question /detail/drawer/index.tsx:9]:
```suggestion
export type QuestionDetailDrawerContentProps = IQuestiohn;
```

이렇게 바꿔보세요!

**Comment by pickk-dev** [comment]
#### 📝Changed routes:


| [**/manage/question/\[id\]**](https://pickk-super-admin-5rkcpn5yr.vercel.app/manage/question/[id]) | [**/manage/question**](https://pickk-super-admin-5rkcpn5yr.vercel.app/manage/question) |
|:-:|:-:|
|<a href="https://pickk-super-admin-5rkcpn5yr.vercel.app/manage/question/[id]">Screenshot of /manage/question/[id]</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/PbuXF1ONJOzblFcLuzUIs75A_kc_.png">(view full size)</a> |<a href="https://pickk-super-admin-5rkcpn5yr.vercel.app/manage/question">Screenshot of /manage/question</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/pOMmCKNG9ENab1pMoMZCdt5wsFg_.png">(view full size)</a> |







Commit 36f39c1b7418950148f5e1d31fdf56b2d31531f2 (https://pickk-super-admin-5rkcpn5yr.vercel.app).

File: src/component/organisms/list-table.tsx [hunk 2]
```diff
@@ -19,15 +19,24 @@ export type ListTableProps = Pick<
...
+    scroll = {},
+    rowSelection = null,
+    onRow = null,
+  }: ListTableProps,
+  ref
+) {
+  useImperativeHandle(ref, () => ({
+    callMutate: () => {
+      mutate();
+    },
```
Comment (greatSumini) [inline src/component/organisms/list-table.tsx:37]:
```
    mutate,
```

File: src/components/question/list/index.tsx [hunk 3]
```diff
@@ -13,6 +13,8 @@ import { QuestionListParams } from '@src/types';
 import { itemColumns } from './columns';
 
 function QuestionListTable() {
+  const listTableRef: any = createRef();
```
Comment (greatSumini) [inline src/components/question/list/index.tsx:16]: useRef를 사용해보세요~

**Pull request merged**